Client/multiplayer_sa/CMultiplayerSA.cpp

```diff
@@ -578,6 +578,76 @@ CMultiplayerSA::CMultiplayerSA()
     m_dwLastStaticAnimID = eAnimID::ANIM_ID_WALK;
 }
 
+namespace GrainEffect
+{
+
+static BYTE ucGrainEnabled = FALSE;
+static DWORD dwGrainStrength = 1.0f;
+
+struct MasterModifier
```

Comment:
Dont mix float and integer multiply without casting first, because it causes warnings.
Please add the necessary casts.

Reply:
The floating point value is a misprint, it should be unsigned int actually.